pom.xml

```diff
@@ -72,7 +72,7 @@
         <not-yet-commons-ssl.version>0.3.11</not-yet-commons-ssl.version>
         <apache-mime4j.version>0.8.1</apache-mime4j.version>
         <log4j.version>2.12.1</log4j.version>
-        <target.jdk>1.6</target.jdk>
+        <target.jdk>1.8</target.jdk>
```

Comment:
Not sure about that change: it needs a vote on mailing list.

Let's decorelate..